src/simulation/openables/BonusOpenables.ts

```diff
@@ -0,0 +1,169 @@
+import { SkillsEnum } from './../../constants';
+export class BonusOpenables {
+	public item: string;
+	public qty: number | [number, number];
+	public weight: number;
+	public skill: SkillsEnum;
+	public req: number | [number, number];
+	public constructor() {
+		this.qty = 0;
+		this.weight = 1;
+		this.req = 1;
+	}
+}
```

Comment:
This class doesnt seem useful, considering it has no methods or anything, Also, youre missing a new line above it, and the name should not be a plural.